docs/source/tutorials/configure_systems.rst

```diff
@@ -215,5 +216,6 @@ Python Environment System
 .. important::
 
     The python environment system can only be used to evaluate onnx models. It must have :code:`onnxruntime` installed!
+    Or Olive can manage the environment by setting :code:`olive_managed_env = True`. This feature works best when used from Conda.
```

Comment:
>olive_managed_env [](http://example.com/codeflow?start=58&length=17)

better give an example including the requirements.txt etc so user could easily copy/paste then modify to use it.